packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [53.2.0]
+
+### Changed
+
+- Adds safeguard for keyring being locked or unlocked ([#5473](https://github.com/MetaMask/core/pull/5473))
```

Comment:
```suggestion
- Check if `KeyringController` is unlocked before processing account events in `MultichainBalancesController` ([#5473](https://github.com/MetaMask/core/pull/5473))
  - This is needed since some Snaps might encrypt their state which needs the `KeyringController` to be unlocked.
```